lightbulb/ext/tasks/triggers.py

```diff
@@ -76,7 +96,23 @@ class CronTrigger(Trigger):
 
         __slots__ = ("_croniter",)
 
-        def __init__(self, crontab: str) -> None:
+        def __init__(
```

Comment:
You should probably add typing overloads for `__init__` considering either crontab or kwargs should be passed.